### ISSUE TITLE
if a user enters the same variable in the debug console and the value hasn't changed, the value isn't announced

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#221965]
1. start debugging and enable a screen reader
2. on a breakpoint, enter a var name in the debug console
3. you'll hear it's value ✔ 
4. without continuing the debug session, enter it again
5. 🐛 screen reader doesn't announce the value

**Comment by meganrogge** [comment]
By enabling `trace` logging, I can see that we're doing the right thing here. The screen reader is choosing not to announce it, so I imagine that's working as designed and might be result of using `aria` status. 

This is an unlikely case for a user because one would usually only check the value after continuing, so closing as there's nothing we can do. cc @rperez030 and @jooyoungseo for any ideas here.

![Image](https://github.com/user-attachments/assets/5e8deee9-de11-437a-9496-540712a94a76)

**Comment by rperez030** [comment]
This is likely screen readers being smart. it is also possible that different screen readers will behave differently. in any case, a screen reader user can always explore the value manually. i wouldn't take any additional action here.

**Comment by hediet** [comment]
@meganrogge is this because the aria alert role attribute does not change (because the new value equals the old value)?

**Comment by meganrogge** [comment]
@hediet yes, probably

**Comment by hediet** [comment]
In that case, can you remove the div element that has the aria alert attribute and re-create it (if it is a container for other stuff)?
@aiday-mar encountered a similar problem for making the monaco editor screenreader-friendly when EditContext is used,

**Comment by rperez030** [comment]
i honestly wonder whether that is even necessary. I would personally take the lack of announcement as an indication that there is no change in value, but maybe others have different expectations. The more we try to engineer the experience the more likely we are to cause unexpected behaviors across the different AT combinations.

**Comment by meganrogge** [comment]
Cool, thanks for the insight. Closing this as designed
